src/edge/recombine.py

```diff
@@ -500,10 +500,12 @@ def lock_genome(genome):
     return genome
 
 
-def recombine_region(genome, region, min_homology_arm_length, op, need_new_fragment):
+def recombine_region(genome, region, min_homology_arm_length, op, is_new_fragment):
     """
     Recombines on a given region. Returns recombination cassette location, how
     many base pairs removed, how many base pairs added, new fragment id.
+    is_new_fragment is a dictionary that indicates if the region.fragment_id is already
```

Comment:
I suggest renaming `is_new_fragment` to something like `new_fragments_dict` to be more readable. `is_new_fragment` sounds like a True Falsy variable.